config.json

```
@@ -788,7 +788,7 @@
         "name": "High Scores",
         "uuid": "f5af74f7-259c-48e6-828c-bcd98717d8dd",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays", "array", "rest-and-spread", "classes"],
```

You probably meant this as second item, I guess:
```suggestion
        "prerequisites": ["arrays", "array-transformations", "rest-and-spread", "classes"],
```

---

Yeah, sorry about that